src/js/components/about/Leadership.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Leads } from './config';
+export default class Leadership extends React.Component {
+  render() {
+    return (
+      <section className="half-section leadership">
+        <div className="container">
+          <div className="row">
+            <h4 className="section-heading">Our Vision</h4>
+              {Leads.map((lead, index) => {
+                return (
+                  <div key={index} className='col-sm-6 col-md-4 lead'>
+                    <h5>{lead.name}</h5>
+                    <p>{lead.position}</p>
+                  </div>
```

Comment:
@andela-gogbara I'll go ahead and make the change with the other stuff I'm doing